src/main/java/seedu/contax/ui/MainWindow.java

```diff
@@ -152,31 +190,24 @@ private void setWindowDefaultSize(GuiSettings guiSettings) {
     }
 
     /**
-     * Changes the type of model being displayed in the content list.
-     *
-     * @param contentType The type of content the UI should display.
+     * Changes the selected tab from the specified {@code contentType}.
```

Comment:
Minor nitpick, this javadoc comment should be `to` instead of `from`. You can follow the javadoc from the before diff

Reply:
Thanks for the feedback. I rewrote the javadoc similar to the one before the diff https://github.com/AY2122S2-CS2103-W17-1/tp/pull/214/commits/1435a8fd7ebd6947196ea01beb903f76afef184f